dev/com.ibm.ws.concurrent.persistent/src/com/ibm/wsspi/concurrent/persistent/TaskStore.java

```diff
@@ -370,4 +388,15 @@
      * @throws Exception if an error occurs when attempting to update the persistent task store.
      */
     int transfer(Long maxTaskId, long oldPartitionId, long newPartitionId) throws Exception;
+
+    /**
+     * Reads information from the partition entry for polling coordination.
```

Comment:
Javadoc should say "Writes..." or "Updates..." instead of "Reads...".

Reply:
Good catch, I'm fixing this and also cleaning up the parameters to align with the designed solution now that there is some more clarity